index.html

```diff
@@ -0,0 +1,236 @@
+            </p>
+          </div>
+          <ul class="SocialMedia">
+            <li>
+              <a href="https://github.com/Robison-faike123">
+                <img src="images/Icon -GitHub.png" alt="-GitHub" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/">
+                <img src="images/Icon - Linkedin.png" alt="Linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Angelist.png" alt="Angelist" />
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com">
+                <img src="images/Icon - Twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Medium.png" alt="Medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <!--this is for second section -->
+    <section class="Works">
+      <div class="RecentWorks">
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="MultiPostStories">
+        <img
+          class="MyResentWorkImg1"
+          src="/images/MyRecentWorkImg1.png"
+          alt=""
+        />
+    
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languageBoxes">
+          <li class="box">
+            <p>CSS</p>
+          </li>
+          <li class="box">
+            <p>HTML</p>
+          </li>
+          <li class="box">
+            <p>BOOTSTRAP</p>
+          </li>
+          <li class="box">
+            <p>RUBY</p>
+          </li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w1">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w2">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
```

Comment:
- [ ] Also, on further inspection, you can consider making the padding in the technologies and the text size smaller so that it might look like the overall card appearance as it is in the Figma design.

**This is how I am seeing the card right now:**

![image](https://user-images.githubusercontent.com/86318284/192718117-15695323-ab09-4643-9ec2-72576d789c0a.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/192718229-20356f5e-af7d-4aa0-83d9-edcda83b5b31.png)

**Kindly ensure you make this change on all the cards**

Reply:
it has been worked on also